src/sidebar/store/test/use-store-test.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+const { mount } = require('enzyme');
+const { createStore } = require('redux');
+const { createElement } = require('preact');
+const { act } = require('preact/test-utils');
+
+const useStore = require('../use-store');
+const { $imports } = useStore;
+
+const initialState = { value: 10, otherValue: 20 };
+const reducer = (state = initialState, action) => {
+  if (action.type === 'INCREMENT') {
+    return { ...state, value: state.value + 1 };
+  } else if (action.type === 'INCREMENT_OTHER') {
+    return { ...state, otherValue: state.otherValue + 1 };
+  } else {
+    return state;
+  }
+};
+
+describe('useStore', () => {
+  let renderCount;
+  let testStore;
+  let TestComponent;
+
+  beforeEach(() => {
+    renderCount = 0;
+
+    // eslint-disable-next-line react/display-name
+    TestComponent = () => {
+      renderCount += 1;
+      const aValue = useStore(store => store.getState().value);
```

Comment:
The name of this state property (`value`) threw me because it sounds like it means something "special" but I think it's just any old property?

Reply:
That's right. It is just a value from the "reducer" function above.